pkg/descheduler/policyconfig.go

```diff
@@ -46,10 +50,150 @@ func LoadPolicyConfig(policyConfigFile string) (*api.DeschedulerPolicy, error) {
 		return nil, fmt.Errorf("failed decoding descheduler's policy config %q: %v", policyConfigFile, err)
 	}
 
-	internalPolicy := &api.DeschedulerPolicy{}
-	if err := scheme.Scheme.Convert(versionedPolicy, internalPolicy, nil); err != nil {
+	// Build profiles
+	// TODO(jchaloup): replace this with v1alpha1 -> v1alpha2 conversion
+	internalPolicy, err := V1alpha1ToInternal(client, versionedPolicy)
+	if err != nil {
 		return nil, fmt.Errorf("failed converting versioned policy to internal policy version: %v", err)
 	}
 
 	return internalPolicy, nil
 }
+
+func V1alpha1ToInternal(
```

Comment:
I renamed this to V1alpha1ToInternal, since we don't actually really have the v1alpha2 type here. I will work it out on the next PRs

Reply:
using an internal type is fine, we may want to even leave it like that if we are planning to rapidly iterate and release more api versions